### ISSUE TITLE
[Bug]: cannot reach the 0.8 EER of ECAPA-TDNN on multi-gpu 

### ISSUE THREAD

**Issue by DONGDILLON** [speechbrain/speechbrain#2056]
### Describe the bug

I have used 8  gpus to train the ECAPA-TDNN and modified  step_size to 8125. However I only get 1 EER with default config parameters. How to train the ECAPA-TDNN and reach 0.8 EER?  Looking forward to your reply

### Expected behaviour

No response

**Comment by underdogliu** [comment]
@DONGDILLON Thanks for your issue! I think such issue theoretically may due to the lower step size which results in too rapid learning rate decrease. But I would like to ask for a bit more information:
- What is your training data? VoxCeleb2 only or VoxCeleb1+2? They may result in different performance.
- Did you re-produce the 0.8% EER inference model using the pre-train models stored at hugging face? If not, please prepare the VoxCeleb trial accordingly and run the inference using `verification_ecapa.py`, to see if the code is running at large-scale properly.

**Comment by DONGDILLON** [comment]
@underdogliu Thanks for your reply. I have noticerd that step size in default config is 65000 which is closed to the step on 1 gpu. However the number of steps on 8 gpu is 8000, so I modified the step size to 8125.
For the first question, I used VoxCeleb2 for training and VoxCeleb1+2 for computing EER following the guidelines in the readme
As for the other question, I have used  pre-train models stored at hugging face and get the 0.84% EER
